### ISSUE TITLE
Can I use this plugin in SPA mode?

### ISSUE THREAD

**Issue by dilovar-91** [TiagoDanin/Nuxt-SEO#84]
Can I use this plugin in SPA mode. does it make sense to use this plugin in SPA

**Comment by TiagoDanin** [comment]
Yes, if you were to need to manipulate the page title or have your SPA (just one page) indexed correctly on Google.

**Comment by TiagoDanin** [comment]
#109
